client/build_test.go

```diff
@@ -82,6 +82,7 @@ func TestClientGatewayIntegration(t *testing.T) {
 }
 
 func testClientGatewaySolve(t *testing.T, sb integration.Sandbox) {
+	integration.SkipIfDockerd(t, sb, "local export")
```

Comment:
local exporter is allowed in moby

Reply:
Ok smth is wrong then: https://github.com/crazy-max/buildkit/runs/5573364411?check_suite_focus=true#step:8:4603

```
=== CONT  TestIntegration/TestExportCacheLoop/worker=dockerd/frontend=client
    dockerfile_test.go:455: 
        	Error Trace:	dockerfile_test.go:455
        	            				run.go:87
        	            				run.go:201
        	Error:      	Received unexpected error:
        	            	unknown cache exporter: "local"
        	            	github.com/moby/buildkit/util/stack.Enable
        	            		/src/util/stack/stack.go:77
        	            	github.com/moby/buildkit/util/grpcerrors.FromGRPC
        	            		/src/util/grpcerrors/grpcerrors.go:188
        	            	github.com/moby/buildkit/util/grpcerrors.UnaryClientInterceptor
        	            		/src/util/grpcerrors/intercept.go:41
        	            	google.golang.org/grpc.(*ClientConn).Invoke
        	            		/src/vendor/google.golang.org/grpc/call.go:35
        	            	github.com/moby/buildkit/api/services/control.(*controlClient).Solve
        	            		/src/api/services/control/control.pb.go:1443
        	            	github.com/moby/buildkit/client.(*Client).solve.func2
        	            		/src/client/solve.go:208
        	            	golang.org/x/sync/errgroup.(*Group).Go.func1
        	            		/src/vendor/golang.org/x/sync/errgroup/errgroup.go:57
        	            	runtime.goexit
        	            		/usr/local/go/src/runtime/asm_amd64.s:1581
        	            	failed to solve
        	            	github.com/moby/buildkit/client.(*Client).solve.func2
        	            		/src/client/solve.go:221
        	            	golang.org/x/sync/errgroup.(*Group).Go.func1
        	            		/src/vendor/golang.org/x/sync/errgroup/errgroup.go:57
        	            	runtime.goexit
        	            		/usr/local/go/src/runtime/asm_amd64.s:1581
        	Test:       	TestIntegration/TestExportCacheLoop/worker=dockerd/frontend=client
```